### ISSUE TITLE
alt attributes for images

### ISSUE THREAD

**Issue by khayyamsaleem** [NoortjeCreemers25/TeaCozy#5]
Do your best to include alt attributes every time you have an image on your page. This "alt text" will serve as a placeholder in the event that your image resources ever fail. 

https://github.com/NoortjeCreemers25/TeaCozy/blob/master/index.html#L35
